.storybook/main.js

```diff
@@ -1,31 +1,33 @@
-const path = require('path');
+const path = require("path");
 
 module.exports = {
-  stories: ['../src/**/*.stories.(js|mdx)', './welcome-page/welcome.stories.js', '../docs/*.stories.mdx'],
+  stories: [
+    "../src/**/*.stories.(js|mdx)",
+    "./welcome-page/welcome.stories.js",
+    "../docs/*.stories.mdx",
+  ],
   addons: [
-    '@storybook/addon-actions/register',
-    '@storybook/addon-knobs/register',
-    '@storybook/addon-notes/register',
-    '@storybook/addon-viewport/register',
-    '@storybook/addon-a11y/register',
-    '@storybook/addon-docs',
-    '@storybook/addon-google-analytics/register',
-    '@storybook/addon-links',
-    './.storybook/theme-selector/register',
+    "@storybook/addon-actions/register",
+    "@storybook/addon-knobs/register",
+    "@storybook/addon-notes/register",
+    "@storybook/addon-viewport/register",
+    "@storybook/addon-a11y/register",
+    "@storybook/addon-docs",
+    "@storybook/addon-google-analytics/register",
+    "@storybook/addon-links",
+    "./.storybook/theme-selector/register",
   ],
   webpackFinal: async (config, { configType }) => {
-    config.module.rules.push(
-      {
-        test: /\.scss$/, use: ['style-loader', 'css-loader', 'sass-loader']
-      }
-    );
-  
+    config.module.rules.push({
+      test: /\.scss$/,
+      use: ["style-loader", "css-loader", "sass-loader"],
```

Comment:
I don't think we need the sass-loader anymore do we?

Reply:
Maybe we don't need any of these loaders even

Reply:
I've opted to leave it as it is, as it's only storybook. You're right, I could leave only one loader.

Reply:
actually we don't need any loader here, so I've removed them all